src/Symfony/Bundle/FrameworkBundle/Templating/Helper/AssetsHelper.php

```diff
@@ -78,7 +78,7 @@ public function getVersion($path = null, $packageName = null)
 
         // packageName is null and path not, so path is a path or a packageName
         try {
-            $package = $this->packages->getPackage($path);
+            $this->packages->getPackage($path);
```

Comment:
was that a bug? why calling getPaclahe if we don't do anything with the return value?